src/main/java/com/minecolonies/entity/EntityCitizen.java

```diff
@@ -203,7 +229,11 @@ private synchronized void updateNavigatorField()
     private void initTasks()
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Move the "com.minecolonies.job.Guard" string literal on the left side of this string comparison. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AS1132)